pmd/pmd-eclipse-plugin
# Resolve deprecations

~Still needed:~

* manual tests :heavy_check_mark: 
* check compatibility with m2e-code-quality plugin :heavy_check_mark: 

See also pmd/pmd#2635
